legacy/editor/TextureGrWnd.cpp

```diff
@@ -615,8 +615,8 @@ void CTextureGrWnd::TexGrStartOpenGL()
 	{
 		app=(oeWin32Application *)Descent;
 		StateLimited=1;
-		save_wnd=(HWND)app->m_hWnd;
-		app->m_hWnd=(HWnd)m_hWnd;
+    save_wnd = app->m_hWnd;
```

Comment:
indent looks broken there, could you run clang-format on your files?

Reply:
Legacy files were not reformatted. Doing what you ask would result in a huge diff.